truss/remote/baseten/core.py

```diff
@@ -234,6 +234,30 @@ def exists_model(api: BasetenApi, model_name: str) -> Optional[str]:
     return model["model"]["id"]
 
 
+def get_model_team_id(api: BasetenApi, model_name: str) -> Optional[str]:
+    """
+    Get the team_id for an existing model.
+
+    Args:
+        api: BasetenApi instance
+        model_name: Name of the model to check
+
+    Returns:
+        team_id if model exists and has a team, otherwise None
```

Comment:
can this return `team_name` and `team_id`? then we can save ourselves a lookup in `model_team_resolver`